independent-publisher-connectors/Care Quaility Comission For England/CQC-Connector.swagger.json

```diff
@@ -0,0 +1,843 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "CQC Data",
+    "description": "Reports & ratings for health & social services regulated by the CQC in England",
+    "version": "10",
+    "contact": {
+      "name": "Martyn Lesbirel",
+      "email": "ipc-author-cqc@outlook.com"
+    }
+  },
+  "host": "api.cqc.org.uk",
+  "basePath": "/public/v1",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/providers": {
+      "get": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
+          }
+        },
+        "summary": "Providers, organisations for health, social care services regulated by the CQC",
```

Comment:
@martynlesbirel All the summary values need to improved as they are vague and doesn't look like operation names.
You can refer [a connector](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/certified-connectors/SAS%20Decisioning/apiDefinition.swagger.json#L37) in our open source repo for reference.

Documentation - [Summaries](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#define-operation-and-parameter-summaries-and-descriptions)